internal/services/service.go

```diff
@@ -14,5 +14,5 @@ type Service interface {
 	fmt.Stringer
 
 	// Starts a service and sends a message to the channel when ready
-	Start(ctx context.Context, ready chan<- struct{}) error
+	Start(ctx context.Context, ready chan<- struct{}, workDir string) error
```

Comment:
I don't like this new parameter on `service.Start()` because the concept doesn't apply to every service, only to `CommandService`. I suggest adding `workDir` as a property of `CommandService` and propagating it downwards from `node.Setup()` to the relevant functions.

The main rationale is that these methods will either die or change in node v2.0 since services will become Go code, no longer Rust executables, but the Service interface will remain and I would like to keep it correct.

Reply:
Done!